pages/common/ssh.md

```diff
@@ -27,6 +27,6 @@
 
 `ssh -L {{9999}}:slashdot.org:80 {{username}}@{{remote_host}}`
 
-- SSH enable agent forward:
+- Forward the authentication information to the remote machine by passing an option (For full list of possible options, try `man ssh_config`):
```

Comment:
I'd shorten this description a bit. How about "Enable the option to forward the authentication information to the remote machine (see `man ssh_config` for available options):"? This has the benefit of following the same order as the elements of the command.